mmtrack/core/collate/collate_enhance.py

```diff
@@ -0,0 +1,16 @@
+from collections.abc import Mapping, Sequence
+
+from mmcv.parallel import collate
+
+
+def collate_enhance(batch, samples_per_gpu=1):
```

Comment:
1. `collate_enhance` is not a good name for a function, "enhance" means nothing.
2. If you indeed need to support new features, make a PR to mmcv to supplement the collate function.
3. This function seems hard code, I think we have more elegant solutions for that.